manifests/y/YutakaSawada/MultiPar/Beta/1.3.2.6/YutakaSawada.MultiPar.Beta.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: YutakaSawada.MultiPar.Beta
+PackageVersion: 1.3.2.6
+PackageLocale: en-US
+Publisher: Yutaka Sawada
+PackageName: MultiPar
+License: Proprietary
```

```suggestion
License: Proprietary
LicenseUrl: https://hp.vector.co.jp/authors/VA021385/archive.htm
```
Found this on the author's page, says the GUI app is closed source. We can link to the license here.

---

Alright.